src/main/java/io/stargate/sgv2/jsonapi/service/processor/CommandProcessor.java

```diff
@@ -57,11 +58,16 @@ public <T extends Command> Uni<CommandResult> processCommand(
         // resolver can be null, not handled in CommandResolverService for now
         .flatMap(
             resolver -> {
-              // if we have resolver, resolve operation and execute
+              // if we have resolver, resolve operation
               Operation operation = resolver.resolveCommand(commandContext, command);
-              return operation.execute(queryExecutor);
+              return Uni.createFrom().item(operation);
             })
 
+        // run on worker thread pool since operation hsa blocking code for vectorize
```

Comment:
since the `resolveCommand ` operation has blocking code for vectorize?

Reply:
Yes